### PR TITLE
Feature/scheduled playwright tests

### PR DESCRIPTION
**Pull request by lilia1891**
Update CI workflow to separate and schedule playwright basic and accessibility tests.
* Added separate jobs for basic and accessibility tests in the ci-e2e.yaml workflow.
* Basic tests (basic.spec.ts) now run daily from Monday to Friday.
* Accessibility tests (accessibility.spec.ts) now run once a month and check only for the critical issues.

### CODE REVIEW

File: .github/workflows/ci-e2e.yaml [hunk 1]
```diff
@@ -82,3 +84,61 @@ jobs:
...
+
+      - name: Running Playwright accessibility tests
+        run: node_modules/.bin/playwright test accessibility.spec.ts
+        env:
+          TEST_PLAN_IDENTIFIERS: ${{ env.TEST_PLAN_IDENTIFIERS }}
+          TEST_PAGE_BASE_URL: ${{ env.FRONTEND_BASE_URL }}
+          APLANS_API_BASE_URL: ${{ env.APLANS_API_BASE_URL }}
+
+      - uses: actions/upload-artifact@v3
+        if: always()
```
Comment (tituomin) [inline .github/workflows/ci-e2e.yaml:140]: It looks like a lot of copy-pasting happened here, not sure if there is a way to refactor these parts and reuse?

**Comment by MWargelin** [comment]
@lilia1891 @tituomin This PR has been stale for a while, is it still relevant?

**Comment by lilia1891** [comment]
@MWargelin I don't think it's relevant, we can delete it

**Comment by MWargelin** [comment]
@lilia1891 Alright, I closed the PR. The branch still exists in case we want to come back to this, it's also always possible to reopen the PR :slightly_smiling_face:

**Pull request closed without merging**